org.siril.Siril.json

```diff
@@ -0,0 +1,288 @@
+{
+    "app-id": "org.siril.Siril",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "command": "siril",
+    "rename-icon": "org.siril.Siril",
+    "rename-mime-file": "siril.xml",
+    "rename-mime-icons": [
+        "text-x-seq"
+    ],
+    "provides": {
+        "id": "org.free_astro.siril"
+    },
+    "replaces": {
+        "id": "org.free_astro.siril"
+    },
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        /* Needed for gvfs to work */
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfs",
```

Comment:
Is this needed ? According to https://docs.flatpak.org/en/latest/sandbox-permissions.html#gvfs-access

> Typical GNOME and GTK applications should use:
> --talk-name=org.gtk.vfs.*
> --filesystem=xdg-run/gvfsd

Reply:
I remember checking back then these were the right one.